### PR TITLE
add icons

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/icons/AllSeeingSealEye.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import classnames, { stroke } from 'classnames/tailwind'
+
+const strokeSecondaryDimmed = classnames(stroke('stroke-secondary-dimmed'))
+const strokeTertiaryLine = classnames(stroke('stroke-tertiary'))
+
+export default function () {
```
Comment (T-Damer) [inline src/icons/AllSeeingSealEye.tsx:6]: Maybe let's call it `SealEye`?

File: src/icons/AllSeeingSealEye.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import classnames, { stroke } from 'classnames/tailwind'
+
+const strokeSecondaryDimmed = classnames(stroke('stroke-secondary-dimmed'))
```
Comment (T-Damer) [inline src/icons/AllSeeingSealEye.tsx:3]: You don't need classnames here

File: src/icons/Dots.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import classnames, { fill } from 'classnames/tailwind'
+
+const fillAccent = classnames(fill('fill-accent'))
```
Comment (T-Damer) [inline src/icons/Dots.tsx:3]: You don't need classnames here

File: src/icons/Eye.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import classnames, { stroke } from 'classnames/tailwind'
+
+const strokeLine = classnames(stroke('stroke-accent'))
```
Comment (T-Damer) [inline src/icons/Eye.tsx:3]: You don't need classnames here

File: src/icons/Logo.tsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
+import classnames, { fill, stroke } from 'classnames/tailwind'
+
+const strokeLine = classnames(stroke('stroke-accent'))
+const strokeSecondaryLine = classnames(stroke('stroke-secondary'))
```
Comment (T-Damer) [inline src/icons/Logo.tsx:4]: You don't need classnames here

File: src/icons/Eye.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+export default function () {
```
Comment (T-Damer) [inline src/icons/Eye.tsx:5]: `OpenEye` or `ClosedEye`? We have both on SealCaster, check design doc

File: src/icons/Logo.tsx [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+        className={fill('fill-accent')}
```
Comment (T-Damer) [inline src/icons/Logo.tsx:128]: Worth extracting into `const`?

File: src/icons/AllSeeingSealEye.tsx [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+      <path
+        d="M27.4123 34.6991C28.791 34.6991 29.9086 34.8644 29.9086 33.283C29.9086 31.7016 28.791 30.4196 27.4123 30.4196C26.0336 30.4196 24.9159 31.7016 24.9159 33.283C24.9159 34.8644 26.0336 34.6991 27.4123 34.6991ZM27.4123 34.6991C27.4123 35.6501 27.4123 37.7661 27.4123 38.622M27.4123 38.622C27.4123 41.475 30.9785 41.4749 30.9785 38.622M27.4123 38.622C27.4123 41.475 23.8461 41.475 23.8461 38.622"
+        className={strokeTertiaryLine}
+        stroke-width="1.5"
+        stroke-linecap="round"
+      />
+      <path
+        d="M38.8244 34.6992L34.5449 35.4125"
+        className={strokeTertiaryLine}
+        stroke-width="1.5"
```
Comment (T-Damer) [inline src/icons/AllSeeingSealEye.tsx:24]: You can extract this into `const`, simply add `strokeWidth: 1.5` into tailwind.config

File: src/icons/AllSeeingSealEye.tsx [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+const strokeTertiaryLine = classnames(stroke('stroke-tertiary'))
+
+export default function () {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="55"
+      height="46"
+      viewBox="0 0 55 46"
+      fill="none"
```
Comment (T-Damer) [inline src/icons/AllSeeingSealEye.tsx:13]: This should be set by default from `index.css` like this:
```CSS
svg:not(.Toastify__toast-icon svg) {
  fill: none;
}
path:not(.walletconnect-qrcode__image path),
circle {
  stroke-linecap: round;
  stroke-linejoin: round;
}
```
So you'll save many lines

File: src/icons/AllSeeingSealEye.tsx [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+        d="M27.4123 34.6991C28.791 34.6991 29.9086 34.8644 29.9086 33.283C29.9086 31.7016 28.791 30.4196 27.4123 30.4196C26.0336 30.4196 24.9159 31.7016 24.9159 33.283C24.9159 34.8644 26.0336 34.6991 27.4123 34.6991ZM27.4123 34.6991C27.4123 35.6501 27.4123 37.7661 27.4123 38.622M27.4123 38.622C27.4123 41.475 30.9785 41.4749 30.9785 38.622M27.4123 38.622C27.4123 41.475 23.8461 41.475 23.8461 38.622"
+        className={strokeTertiaryLine}
+        stroke-width="1.5"
+        stroke-linecap="round"
+      />
+      <path
+        d="M38.8244 34.6992L34.5449 35.4125"
+        className={strokeTertiaryLine}
+        stroke-width="1.5"
+        stroke-linecap="round"
```
Comment (T-Damer) [inline src/icons/AllSeeingSealEye.tsx:25]: Please remove all `stroke-linecap="round"` and set them from index.css:
```CSS
path:not(.walletconnect-qrcode__image path),
circle {
  stroke-linecap: round;
  stroke-linejoin: round;
}
```

File: src/icons/AllSeeingSealEye.tsx (hunk 2 above)
Comment (T-Damer) [reply src/icons/AllSeeingSealEye.tsx:3]: But if you specify `strokeWidth('stroke-1.5')`, then you'll need it, take a look below

File: src/icons/Dots.tsx [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+export default function () {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="36"
+      height="6"
+      viewBox="0 0 36 6"
+      fill="none"
+    >
+      <circle cx="3" cy="3.00024" r="3" className={fillAccent} />
```
Comment (T-Damer) [inline src/icons/Dots.tsx:17]: Seems like you can do `cy="3" r="3"`

Comment (T-Damer) [reply src/icons/Dots.tsx:17]: It shouldn't affect the way it looks like

File: src/icons/Dots.tsx [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+const fillTertiary = classnames(fill('fill-tertiary'))
+
+export default function () {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="36"
+      height="6"
+      viewBox="0 0 36 6"
+      fill="none"
```
Comment (T-Damer) [inline src/icons/Dots.tsx:15]: You don't need this if you add styles into `index.css`

File: src/icons/Eye.tsx [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+      xmlns="http://www.w3.org/2000/svg"
+      width="19"
+      height="13"
+      viewBox="0 0 19 13"
+      fill="none"
+    >
+      <path
+        d="M17.348 5.54566C17.7795 6.11815 17.7795 6.88185 17.348 7.45434C16.161 9.02918 13.4044 12 9.5 12C5.59559 12 2.83899 9.02918 1.652 7.45434C1.22051 6.88186 1.22051 6.11815 1.652 5.54566C2.83899 3.97082 5.59559 1 9.5 1C13.4044 1 16.161 3.97082 17.348 5.54566Z"
+        className={strokeLine}
+        stroke-width="1.5"
```
Comment (T-Damer) [inline src/icons/Eye.tsx:17]: You know what to do

File: src/icons/Dots.tsx [hunk 14]
```diff
@@ -0,0 +1,23 @@
+import classnames, { fill } from 'classnames/tailwind'
+
+const fillAccent = classnames(fill('fill-accent'))
+const fillSecondary = classnames(fill('fill-secondary'))
```
Comment (smyzin) [inline src/icons/Dots.tsx:4]: `const fillSecondary = fill('fill-secondary')` and so on

**Review by upacyxou — APPROVED** [review]
lgtm

**Pull request merged**